history/history-network.md

```diff
@@ -236,6 +233,47 @@ content          = SSZ.serialize(block_header_with_proof)
 content_key      = selector + SSZ.serialize(block_number_key)
 ```
 
+##### Ephemeral Block Headers
+
+This content type represents block headers *near* the HEAD of the chain.  They are provable by tracing through the chain of `header.parent_hash` values.  All nodes in the network are assumed to store some amount of this content.  The `Ping.custom_data` and `Pong.custom_data` fields can be used to learn the number of recent headers that a client makes available.
```

Comment:
Should we add a phrase stating that nodes should aim to store at least the latest 8192 ephemeral block headers? (Obviously when syncing they can have less).